angular-link-diver/src/app/app.component.html

```diff
@@ -0,0 +1,5 @@
+<h1>Hello World!</h1>
+
+
+
+<router-outlet></router-outlet>
```

Comment:
When possible, please add a new line for every file for consistency. I think we can configure eslint to enforce it https://eslint.org/docs/rules/eol-last

Please do the same for other files without newlines at the end of file.